### DISCUSSION TITLE
Use Tanstack Table on a simple HTML page by including URL to javascript File ( CDN )

### DISCUSSION THREAD

**Discussion by Er4twXz** [TanStack/table#5391] [Q&A]
Is it possible to use the latest version of tanstack table via url ?

see also: https://github.com/TanStack/table/issues/182

Actually my simple goal is to create a htmlpage showing the data of a json file like
https://github.com/shahnewazshamim/json-to-table

without any react, nodejs, npm --- only simple plain javascript

**Comment by ducknificient** [comment]
hi, any update on this ?

**Comment by bburns** [comment]
I think it might be possible - eg this works, using jsdelivr's cdn - 

```
<script type="module">
  import { createTable } from 'https://cdn.jsdelivr.net/npm/@tanstack/table-core@8.5.15/build/esm/index.js'
</script>
```

To see what's in the cdn - https://cdn.jsdelivr.net/npm/@tanstack/table-core@8.5.15/

A relatively simple example (using vite) - https://tanstack.com/table/latest/docs/framework/vanilla/examples/basic

That's as far as I've taken it - I think I'll use vite instead...
